### PR TITLE
Add Logi Options+ 0.91.3227

### PR DESCRIPTION
**Pull request by nilsstreedain**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

**Note: _"Logi Options+"_ and _"Logitech Options"_ are two completely separate Logitech applications.**

### CODE REVIEW

**Comment by nilsstreedain** [comment]
In the time between creating the pull request and the workflow being approved, Logitech updated the installer to require user interaction which I believe is what caused the workflow to timeout. I have updated the cask to use manual installation to fix the issue.

File: Casks/logi-options-plus.rb [hunk 1]
```diff
@@ -13,41 +13,43 @@
...
+            delete:    [
+              "/Applications/logioptionsplus.app",
+              "/Library/Application Support/Logitech.localized",
+              "/Library/LaunchDaemons/com.logi.optionsplus.updater.plist",
+              "/Library/LaunchAgents/com.logi.optionsplus.plist",
+              "/private/var/folders/pm/x750ngd17hgc1wc7_09wjgdh0000gn/C/com.logi.optionsplus.helper",
+              "/private/var/folders/pm/x750ngd17hgc1wc7_09wjgdh0000gn/T/com.logi.optionsplus.agent.logs",
+              "/private/var/folders/pm/x750ngd17hgc1wc7_09wjgdh0000gn/T/com.logi.optionsplus.installer.logs",
+              "/private/var/folders/pm/x750ngd17hgc1wc7_09wjgdh0000gn/T/com.logi.optionsplus.updater.logs",
+              "/private/var/folders/pm/x750ngd17hgc1wc7_09wjgdh0000gn/T/.com.logi.optionsplus.6N7HiI",
```
Comment (bevanjkay) [inline Casks/logi-options-plus.rb:41]: @nilsstreedain Please remove these files from the uninstall stanza, they are unique to your machine and won't work globally.

File: Casks/logi-options-plus.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "com.logi.optionsplus",
+    "com.logi.optionsplus.updater",
+    "com.logi.optionsplus.agent",
+  ],
+            quit:      [
+              "com.logi.cp-dev-mgr",
+              "com.logi.optionsplus",
+              "com.logi.optionsplus.updater",
+              "com.logi.optionsplus.agent",
+              "com.Logitech.Unifying Software.pkg",
```
Comment (bevanjkay) [inline Casks/logi-options-plus.rb:26]: @nilsstreedain Do you know if this package is specific to `Logitech Options+` or is it required for other Logitech software?

Comment (nilsstreedain) [reply Casks/logi-options-plus.rb]: It appears the `com.LogitechUnifying Software.pkg` may be used in other software so it has been removed

File: Casks/logi-options-plus.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "com.logi.optionsplus.agent",
+  ],
+            quit:      [
+              "com.logi.cp-dev-mgr",
+              "com.logi.optionsplus",
+              "com.logi.optionsplus.updater",
+              "com.logi.optionsplus.agent",
+            ],
+            pkgutil:   [
+              "com.Logitech.Updater.pkg",
```
Comment (bevanjkay) [inline Casks/logi-options-plus.rb:28]: @nilsstreedain I'm assuming the updater is going to be a global Logitech pkg also?

Comment (nilsstreedain) [reply Casks/logi-options-plus.rb]: Ah yes, I missed that, it has been corrected

**Pull request merged**